lib/tasks/high_charts.rake

```diff
@@ -111,7 +113,15 @@ namespace :highcharts do
 
       sh "curl -# http://code.highcharts.com/maps/modules/map.js -L --compressed -o lib/daru/view/adapters/js/highcharts_js/map.js"
       sh "curl -# http://code.highcharts.com/mapdata/countries/in/custom/in-all-andaman-and-nicobar.js -L --compressed -o lib/daru/view/adapters/js/highcharts_js/mapdata/countries/in/custom/in-all-andaman-and-nicobar.js"
+      sh "curl -# http://code.highcharts.com/maps/js/modules/map.js -L --compressed -o lib/daru/view/adapters/js/highcharts_js/js/map.js"
     end  
   end
 
+  task :css do
+    say "Grabbing CSS of HighCharts from Upstream..." do
```

Comment:
Please test it manually once again properly and confirm.

Reply:
I have tested the rake task and the examples again. Everything is working fine.